ql/src/java/org/apache/hadoop/hive/ql/ddl/DDLUtils.java

```diff
@@ -219,4 +220,21 @@ private static String getHS2Host(HiveConf conf) throws SemanticException {
 
     throw new SemanticException("Kill query is only supported in HiveServer2 (not hive cli)");
   }
+
+  /**
+   * Get the fully qualified name in the node.
+   * E.g. the node of the form ^(DOT ^(DOT a b) c) will generate a name of the form "a.b.c".
+   */
+  public static String getFQName(ASTNode node) {
```

Comment:
I don't believe this method will work correctly for quoted cases like:
```
`db`.`table` 
```
right now I don't see the original place where this was...
...it work just like before?

Reply:
The original code for this was not removed yet, as it is still used by some other analyzers which are still in DDLSemanticAnalyzer, see the static class QualifiedNameUtil.  I don't see why wouldn't it work in quoted case, I guess the quotes are removed before building the ASTTree, isn't that the case?